### PR TITLE
Update Strategy

### LINKED ISSUE #63: Update Strategy
The strategy is changing to include streams and hasn't had the time to focus on management support. So the documentation on this should be updated.

### PR DESCRIPTION
**Pull request by s-laugh**
Updates to the strategy, how things are done and the means to which we do them.

Closes #63

### CODE REVIEW

**Comment by s-laugh** [comment]
@ahmad-shahid @GoCAlainBelleville @PhilippeTurcotteGov @jeanbenoitrochon 
Just a draft right now, but I'd like to get a preliminary review on the actual content/text make sure you all agree with it and that it makes sense.

File: strategy/overview.md [hunk 1]
```diff
@@ -1,39 +1,48 @@
...
 
-Each month we will focus on one core [topic](https://github.com/esdc-devcop/esdc-devcop.github.io/issues?q=is%3Aopen+is%3Aissue+label%3Atopic) that could be actioned and implemented by a team with less than half a day's work.
-Each month will build on the previous where it will lead teams to developing applications better and faster though increasing automation and reducing feedback loops.
+## Mission
 
-Throughout all of this, we will need to keep in mind that their are many types of technology being used in applications that will need these implementations;
-So our solutions need to be non-tool dependant but offer suggestions and recommendations for tools.
-We also need to keep in consideration that there are ~140 development teams in IITB and we will not have the capacity to help them all one-by-one.
-So we will need to lay out plans that can be adopted without our intervention.
+Equipping ESDC developers with skills, knowledge and tools to continously modernise the software they develop.
```
Comment (jeanbenoitrochon) [inline strategy/overview.md]: continuously

File: strategy/overview.md [hunk 2]
```diff
@@ -1,39 +1,48 @@
...
 
-We will be providing support to teams that are keen on implementing these topics but are receiving resistance from management.
-Working with management on what they perceive to be preventing them from accepting the implementation to demonstrate how they can do it now.
+## How
 
-## Community Building & Engagement
+### Sessions
 
-We will also focus on keeping the community engaged and involved in the various activities through social media and advertising methods.
+The sessions is currently our main point of contact with the Development Community where we bring updates on the available tooling and direction of development for the department.
```
Comment (jeanbenoitrochon) [inline strategy/overview.md]: "The sessions is" should be "sessions are"

File: strategy/overview.md [hunk 3]
```diff
@@ -1,39 +1,48 @@
...
+
+We want to bring [topics](https://github.com/esdc-devcop/esdc-devcop.github.io/issues?q=is%3Aopen+is%3Aissue+label%3Atopic) the community wants to hear about, so we have an **[open submission](https://github.com/esdc-devcop/esdc-devcop.github.io/issues/new?assignees=&labels=topic&template=potential-devcop-topic.md&title=Topic%3A+) and voting (:+1:) on suggested [topics](https://github.com/esdc-devcop/esdc-devcop.github.io/issues?q=is%3Aopen+is%3Aissue+label%3Atopic)**.
+
+We understand that we have a diverse audience, so we have two 1 hour sessions every month.
+The first being an updates and overview on a general [topic](https://github.com/esdc-devcop/esdc-devcop.github.io/issues?q=is%3Aopen+is%3Aissue+label%3Atopic) with an open discussion.
+The second being an in-depth dive into the underlying workings of the [topic](https://github.com/esdc-devcop/esdc-devcop.github.io/issues?q=is%3Aopen+is%3Aissue+label%3Atopic) and allowing for questions on how to really utilize the idea within existing solutions.
+
+### Recommendations & Guides
+
+We are commited to being open to requests for new and updated recommendations and guides on anything the community wants more details on.
```
Comment (jeanbenoitrochon) [inline strategy/overview.md]: committed

**Pull request merged**